src/oop_ext/interface/_interface.py

```diff
@@ -571,9 +636,6 @@ def GetInterfaceMethodsAndAttrs(self, interface):
         try:
             cache = self.cache
         except AttributeError:
-            # create it on the 1st access
-            from oop_ext.foundation.cached_method import ImmutableParamsCachedMethod
-
```

Comment:
_interface.py:642: error: Returning Any from function declared to return "Tuple[Dict[str, Any], Dict[str, Any]]"

Reply:
? Both CI is passing and I don't see any errors locally.

(Please be more specific... just posting some random output without context is not productive).

Reply:
Sure, maybe I forgot to add another comment suggesting this.
Since we are typing a lib that is intended to be generic, we should add more checks for mypy like:

` --disallow-any-decorated`:  which in my view is important (brought to attention bugs like the previous PR).
`--warn-return-any`:  warns when returning a value with type Any from a function declared with a non-Any return type.

I think that we could go further by restricting more and avoiding generics types with Any when possible, but we could do that in other PR's.

Reply:
OK, now the context makes the rest of the comments make more sense. 👍